components/pipedrive/actions/add-activity/add-activity.mjs

```diff
@@ -76,148 +43,159 @@ export default {
+        "leadId",
       ],
-      description: "ID of the person this activity will be associated with",
     },
-    participants: {
-      type: "string[]",
-      label: "Participants",
-      description: "List of multiple persons (participants) this activity will be associated with. If omitted, single participant from `person_id` field is used. It requires a structure as follows: `[{\"person_id\":1,\"primary_flag\":true}]`",
-      optional: true,
+    orgId: {
       propDefinition: [
         pipedriveApp,
-        "personId",
+        "organizationId",
       ],
+      description: "ID of the organization this activity will be associated with",
     },
-    organizationId: {
+    projectId: {
       propDefinition: [
         pipedriveApp,
-        "organizationId",
+        "projectId",
       ],
       description: "ID of the organization this activity will be associated with",
     },
-    note: {
+    dueDate: {
       type: "string",
-      label: "Note",
-      description: "Note of the activity (HTML format)",
+      label: "Due Date",
+      description: "Due date of the activity. Format: `YYYY-MM-DD`",
       optional: true,
     },
-    location: {
+    dueTime: {
       type: "string",
-      label: "Location",
-      description: "The address of the activity. Pipedrive will automatically check if the location matches a geo-location on Google maps.",
+      label: "Due Time",
+      description: "Due time of the activity in UTC. Format: `HH:MM`",
       optional: true,
     },
-    publicDescription: {
+    duration: {
       type: "string",
-      label: "Public Description",
-      description: "Additional details about the activity that will be synced to your external calendar. Unlike the note added to the activity, the description will be publicly visible to any guests added to the activity.",
+      label: "Duration",
+      description: "Duration of the activity. Format: `HH:MM`",
       optional: true,
     },
-    busyFlag: {
+    busy: {
       type: "boolean",
-      label: "Busy Flag",
+      label: "Busy",
       description: "Set the activity as 'Busy' or 'Free'. If the flag is set to true, your customers will not be able to book that time slot through any Scheduler links",
       optional: true,
     },
+    done: {
+      type: "boolean",
+      label: "Done",
+      description: "Whether the activity is done or not.",
+      optional: true,
+    },
+    location: {
+      type: "object",
+      label: "Location",
+      description: "The address of the activity. Pipedrive will automatically check if the location matches a geo-location on Google maps.",
+      optional: true,
+    },
+    participants: {
+      type: "string[]",
+      label: "Participants",
+      description: "List of multiple persons (participants) this activity will be associated with. If omitted, single participant from `person_id` field is used. It requires a structure as follows: `[{\"person_id\":1,\"primary_flag\":true}]`",
```

Comment:
The example `participants` structure should match the structure in the component.
```suggestion
      description: "List of multiple persons (participants) this activity will be associated with. If omitted, single participant from `person_id` field is used. It requires a structure as follows: `[{\"person_id\":1,\"primary\":true}]`",
```